features/transform-box.yml

```
@@ -0,0 +1,10 @@
+name: transform-box
+description: The `transform-box` CSS property sets the position and dimensions, or reference box, relative to which transformations are calculated.
```

```suggestion
description: The `transform-box` CSS property sets the position and dimensions of the reference box relative to which an element's individual transformations are calculated.
```

I was having trouble parsing the initial sentence. Maybe I misunderstood what you meant.
I added the word "individual" in there in an attempt to make it clear that this only applies for individual transform properties (if, indeed, this is how this works).

---

I liked the phrasing, but removed the word `individual`- it applies to the individual transform properties (scale, translate, rotate) as well as `transform`.